docs/source/configurable.rst

```diff
@@ -0,0 +1,260 @@
+Configuring ``garak``
+=====================
+
+Beyond the standard CLI options, garak is highly configurable.
+You can use YAML files to configure a garak run, down to the level
+of exactly how each plugin behaves.
+
+
+Specifying custom configuration
+-------------------------------
+
+Garak can be configured in multiple ways:
+
+* Via command-line parameters
+* Using YAML configs
+* Through specifying JSON on the command line
```

Comment:
The cli json format is not covered here, consider link to the rest documentation json for format of
`--*_option_file` passed values or and added section after the `yaml` details.

Maybe a section `Using a custom json config`?